paymentsheet/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/ach/USBankAccountFormViewModel.kt

```diff
@@ -98,11 +92,11 @@ internal class USBankAccountFormViewModel @Inject internal constructor(
         .stateIn(viewModelScope, SharingStarted.Lazily, args.formArgs.showCheckbox)
 
     val requiredFields = combine(
-        nameElement.getFormFieldValueFlow().map { formFieldsList ->
-            formFieldsList.firstOrNull()?.second?.value?.isNotBlank() ?: false
+        nameController.formFieldValue.map { formFieldEntry ->
+            !formFieldEntry.value.isNullOrBlank()
```

Comment:
I would recommend using formFieldEntry.isComplete, or commenting why it cannot be used here.